tour_model_eval/get_plot.py

```diff
@@ -0,0 +1,54 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from matplotlib import cm
+import folium
+import branca.colormap as clm
+
+
+def get_scatter(percentage,homo_score,valid_users):
+    x=percentage
+    y=homo_score
+    v=valid_users
+    cmp = cm.get_cmap('Dark2', len(valid_users))
+
+    sc = []
+    for i in range(len(valid_users)):
+        for n in range(len(x[i])):
+            point = plt.scatter(x[i][n], y[i][n], color=cmp.colors[i], s=70, alpha=0.7)
+        sc.append(point)
```

Comment:
I don't understand what you mean by using a dataframe or a numpy array here. Since we will have several points for a user, in order to make one legend for one user, I need to collect one of the several points of that user, and `i` here is to control the color. I don't know how to use  a numpy array to do so.

Reply:
did you try to search for "pandas dataframe scatter plot"?
One of the first hits is https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.plot.scatter.html

Reply:
Please document that this needs to be cleaned up later (https://github.com/e-mission/e-mission-docs/issues/631)